CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
+
+1. Build everything and run tests to learn how to do both:
+  - `npm run build`
```

Comment:
Thank you for creating this file.

`npm run build` maybe better be replaced with `npm run build:dist` to run JS build only.

Reply:
I do have a problem with `build:docs` not passing locally so I think it is better to recommend `build:dist`, yes. 

BTW I need to bypass commit hooks, too, because of the problem with `build:docs`. I will try to figure out what the problem with it is so that the commit hooks work locally.

Reply:
Actually I reverted this back to the full `build` command. It runs as a part of the pre-commit hook so all contributors should get it to work. I had problems with it so I dug into what was causing them and the reason was the `sed` invocations were not written in a way that was portable to BSD `sed` used by macOS (what I am using locally). I tried to fix this in #615 but we'll see if in the process of "fixing" it I broke it for GNU sed or not.